Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Coordinate/GetByStreetcodeId/GetCoordinatesByStreetcodeIdHandler.cs

```diff
@@ -19,6 +19,12 @@ public GetCoordinatesByStreetcodeIdHandler(IRepositoryWrapper repositoryWrapper,
 
         public async Task<Result<IEnumerable<StreetcodeCoordinateDTO>>> Handle(GetCoordinatesByStreetcodeIdQuery request, CancellationToken cancellationToken)
         {
+            if ((await _repositoryWrapper.StreetcodeRepository.GetFirstOrDefaultAsync(s => s.Id == request.streetcodeId)) is null)
+            {
+                return Result.Fail(
+                    new Error($"Cannot find a coordinates by a streetcode id: {request.streetcodeId}, because such streetcode doesn`t exist"));
```

Comment:
Let's use localized exception messages.
https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-localized-exception-messages
https://learn.microsoft.com/en-us/dotnet/core/extensions/resources